src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -917,10 +918,45 @@ private static Win32Program GetProgramFromPath(string path)
             }
         }
 
+        private static bool TryGetIcoPathForRunCommandProgram(Win32Program program, out string icoPath)
+        {
+            if (program.AppType == ApplicationType.RunCommand)
```

Comment:
i amsure this should be a not condition

Reply:
Oh, that is embarrassing!
(Never add code after you have tested, without retesting. 😞)

Reply:
Resolved and retested.

![image](https://user-images.githubusercontent.com/36843930/189169896-3c87bab2-db62-41e8-ac63-24b8d4403799.png)

![image](https://user-images.githubusercontent.com/36843930/189170022-98e575cd-ecee-4010-bb50-0cdf0f36c0a4.png)